analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -172,3 +175,73 @@ class Analyzer(private val config: AnalyzerConfiguration, private val labels: Ma
         return analyzerResultBuilder.build()
     }
 }
+
+private class PackageManagerRunner(
```

Comment:
IIUC, this class will become a quite central component in the `Analyzer`. Can you please add some documentation?

Reply:
I added only a short description, because I think the documentation of the constructor arguments explains the rest.